### ISSUE TITLE
🧑‍💻: 

### ISSUE THREAD

**Issue by shrek-28** [Avdhesh-Varshney/AI-Code#92]
:red_circle: **Title** : Enhancement of Home Page
:red_circle: **Enhancement Aim** : Enhancing the look of the homepage so that people know what is the website's intention
:red_circle: **Changes** : 
1. Add a description
2. Add a navbar directing to different sections
3. Give a basic idea of what the website covers

### Screenshots 📷
N/A

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- What is your participant role? SWOC

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Avdhesh-Varshney** [comment]
@shrek-28 I've assigned you. But PR will be count from 1 Jan, 2025 in SWOC

**Comment by shrek-28** [comment]
yes, i will do PR on Jan 1, 2025 itself

**Comment by Avdhesh-Varshney** [comment]
Since website is build using MK Docs library.
So, you have to follow the instructions on README file and [link to homepage](https://github.com/Avdhesh-Varshney/AI-Code/blob/main/docs/index.md).

**Comment by shrek-28** [comment]
can it be moved into proper website based on either simple JS or React framework?

**Comment by Avdhesh-Varshney** [comment]
> can it be moved into proper website based on either simple JS or React framework?

No, Mk docs is simple to contribute for newbie. You just design a homepage and provide the usefulness info. of this project. You can connect me on discord to discuss about the project for effective contribution or raise a github discussion channel

**Comment by shrek-28** [comment]
#94

**Comment by shrek-28** [comment]
Hi, 
Since I have enhanced the Homepage, but my PR has been closed, with zero feedback and explanations, I take my hands off this issue. Please reassign.

**Comment by Avdhesh-Varshney** [comment]
> Hi, Since I have enhanced the Homepage, but my PR has been closed, with zero feedback and explanations, I take my hands off this issue. Please reassign.

![image](https://github.com/user-attachments/assets/5bcb56b8-d648-4319-8be3-2d48cef4253f)


- ***I THINK THERE IS SOME INTERNET CONNECTIVITY ISSUE FROM YOUR END***
